### ISSUE TITLE
 resolved by kernel PR59

### ISSUE THREAD

**Issue by nilesh-dryad** [jefftenney/LPTIM-Tick-U5#1]
I saw this comment in READ ME. I don't get it.
LED is blinking twice particularly in that case. That means error in default tickless mode. Are this comment and error related?

**Comment by jefftenney** [comment]
Yes they are related.  There are some minor errors in the default tickless mode.  Those errors will cause the LED to blink twice in "Test 3".  Those errors are resolved in kernel PR #59.
